scripts/ci-e2e.sh

```diff
@@ -90,6 +90,14 @@ cleanup() {
 
   ps -ef > "${ARTIFACTS_LOCAL}/processes-ps-ef.txt" || true
 
+  for PID in $(ps -eo pid=); do
+    echo "> PID=$PID"
+    echo ">> /proc/${PID}/status" 
+    cat "/proc/${PID}/status" || true
+    echo ">> /proc/${PID}/stack" 
+    cat "/proc/${PID}/stack" || true
+  done >> "${ARTIFACTS_LOCAL}/processes-proc-information.txt"
```

Comment:
On my machine that is 1,4 MB of data. I assume it's significantly less in the Pod at this time, but let's keep an eye on it

Reply:
Could have just checked in e2e-main... 12483 bytes seems okay

Reply:
we could also compress the data. But yes, should be way less due to way less pids inside the prowjob pod.